sdk/resourcemanager/Proto.Client/compute/Convenience/VirtualMachineScaleSetModelBuilderBase.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Azure.ResourceManager.Core;
+using System;
+using System.Collections.Generic;
+
+namespace Proto.Compute.Convenience
+{
+    /// <summary>
+    /// A class representing a builder object to help create a virtual machine scale set.
+    /// </summary>
+    public abstract class VirtualMachineScaleSetModelBuilderBase : VirtualMachineScaleSetBuilder
```

Comment:
unclear what the difference between a VirtualMachineScaleSetBuilder is and a VirtualMachineScaleSetModelBuilder

Reply:
Flattened to partial class with one generated and one to provide customized extension methods.